starter-core/src/main/java/io/micronaut/starter/feature/database/DataMongo.java

```diff
@@ -51,39 +43,20 @@ public String getMicronautDocumentation() {
         return "https://micronaut-projects.github.io/micronaut-data/latest/guide/#mongo";
```

Comment:
should this method be in the parent class? Both implementations override with the same value.

Reply:
Done